doc/theming.rst

```diff
@@ -116,6 +116,12 @@ These themes are:
 
   - **sidebarwidth** (an integer): Width of the sidebar in pixels.  (Do not
     include ``px`` in the value.)  Defaults to 230 pixels.
+  
+  - **canonical_url** (sting): A `canonical url`_ to let search engines know
```

Comment:
How about `canonical_baseurl` instead? I believe this is a part of URL, So `baseurl` is better for me.

Reply:
The reason that I used `canonica_urll` is that a couple 3rd party themes already have this feature and use that variable instead so it will be easier for them to preserve compatibility without having to leave there own variable in their theme.